lib/experimental/Widgets/Content/CalendarEvent/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { DateAvatar } from "@/experimental/Information/Avatars/DateAvatar"
+import { RawTag } from "@/experimental/Information/Tags/RawTag"
+import { Tooltip } from "@/experimental/Overlays/Tooltip"
+import { ChevronRight } from "@/icons/app"
+import { forwardRef } from "react"
+
+type Tag = {
+  icon: IconType
+  label: string
+  description: string
+}
+
+export interface CalendarEventProps {
+  label?: string
```

Comment:
what is _label_? how is it different from _title_?

Reply:
That is the label

<img width="410" alt="Screenshot 2024-12-17 at 10 55 17" src="https://github.com/user-attachments/assets/083c14ff-146e-46bd-b96c-8048cc1fa525" />